### PR TITLE
initial upload

### PR DESCRIPTION
**Pull request by artivis**
Split the snap packaging from the sources originally hosted [here](https://github.com/ubuntu-robotics/ros-foxglove-bridge/tree/feature/snap).

No change but the part's source from local to git remote.

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+  reset-config:
+    command: usr/bin/reset_config.sh
+
+parts:
+  foxglove-bridge:
+    plugin: colcon
+    colcon-cmake-args:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_TESTING=OFF
+    source: https://github.com/foxglove/ros-foxglove-bridge.git
```
Comment (giusebar) [inline snap/snapcraft.yaml:47]: so now we are using upstream instead of our fork?

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -0,0 +1,232 @@
+name: foxglove-bridge
+adopt-info: foxglove-bridge
+license: MIT
```
Comment (giusebar) [inline snap/snapcraft.yaml:3]: should we add a version?

File: .github/workflows/promote.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+name: promote
```
Comment (giusebar) [inline .github/workflows/promote.yaml:1]: how is this workflow triggered?

File: snap/snapcraft.yaml (hunk 1 above)
Comment (artivis) [reply snap/snapcraft.yaml:47]: Yep, there is no reason to use a fork beside hosting the snapcraft.yaml file.

File: snap/snapcraft.yaml (hunk 2 above)
Comment (artivis) [reply snap/snapcraft.yaml:3]: The version is handled in the `foxglove-bridge` part through this `adopt-info` keyword.

File: .github/workflows/promote.yaml (hunk 3 above)
Comment (artivis) [reply .github/workflows/promote.yaml:1]: Manually, as intended. One should make sure that the `candidate` works as expected and manually trigger the promotion from `candidate` to `stable`.

Comment (giusebar) [reply .github/workflows/promote.yaml:1]: ok, cool feat :+1:

**Review by giusebar — APPROVED** [review]

**Review by giusebar — APPROVED** [review]

**Pull request merged**